src/js/modules/ContactAddress.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+//TODO: import correct svg
```

Comment:
https://github.com/encharm/Font-Awesome-SVG-PNG/tree/master/black/svg

Reply:
Thank you